cmd/blockchaincmd/add_validator.go

```diff
@@ -456,6 +458,23 @@ func CallAddValidator(
 	if err != nil {
 		return err
 	}
+	logLvl, err := logging.ToLevel(aggregatorLogLevel)
+	if err != nil {
```

Comment:
nit: we should display user error message and that logLevel is set to debug 
pls add something like  `ux.Logger.PrintToUser("undefined logLevel. Setting signature aggregator log to DEBUG")` or something similar.
pls feel free to ignore this suggestion